app/views/recipes/index.html.haml

```diff
@@ -1,24 +1,21 @@
 .page-header
-  %h1.text-uppercase Recipes
+  %h1.text-uppercase.text-center Recipes
 
-%table.table.table-condensed.table-responsive
-  %thead
-    %tr
-      %th Name
-      %th Preparation
-      %th Time
-      %th Cost
-      %th
-  %tbody
-    - @recipes.each do |recipe|
-      %tr
-        %td= link_to recipe.name, recipe
-        %td= recipe.preparation
-        %td= recipe.preparation_time
-        %td= number_to_currency recipe.cost
-        %td.text-right
-          = link_to 'Edit', edit_recipe_path(recipe), class: 'btn btn-sm btn-primary'
-          = link_to 'Destroy', recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-sm btn-danger'
+.row
+  - @recipes.each_slice(3) do |recipe_two|
+    -recipe_two.each do |recipe|
+      .col-md-4.col-md-4.col-md-4.text-center
+        %div.img-thumbnail.center-block{style:"width:250px; height:200px; border:2px solid #000; margin-bottom:15px; background-color:#eee"}
+        .row
+          .col-md-6.col-md-offset-2.text-left
+            %p Name: #{link_to recipe.name, recipe}
+            %p Time Taken: #{recipe.preparation_time} hours
```

Comment:
@arbins hmm in that case let's keep it simple and let's add hour or minutes in the text field.